nprapps/elections-bots
# chore: organize files

## Describe your changes

1. Organized code in their own files
2. Added a function to delete all data from Google Sheets
3. Added a function to add date to the Google Sheets

## Issue ticket number and link
Ongoing work for [#165](https://github.com/nprapps/elections24-primaries/issues/165)

## Testing Steps

[Google Sheets
](https://docs.google.com/spreadsheets/d/1LFtG54Bmzz_AptuHzdxFbQpH7DaWfvFoMog4WBBIa2k/edit#gid=0)
- Run `node elex-testing-results/deleteDataInSheets.js` to delete data added from the Google Sheets
- Run `node elex-testing-results/index.js` to see data added to the Google Sheets
